eb_hooks.py

```diff
@@ -684,7 +684,8 @@ def post_sanitycheck_cuda(self, *args, **kwargs):
     Remove files from CUDA installation that we are not allowed to ship,
     and replace them with a symlink to a corresponding installation under host_injections.
     """
-    if self.name == 'CUDA':
+    # Make sure we only do this for CUDA and only if we are doing a CVMFS installation
+    if self.name == 'CUDA' and self.installdir.startswith('/cvmfs/software.eessi.io/versions'):
```

Comment:
Hmm, this might not be enough of a guard, if we do `--module-only` I think this would still trigger

Reply:
sanity check is also run when using `--module-only`

Reply:
I guess I will just check the build options for `--sanity-check-only` and `--module-only`

Reply:
Fixed in https://github.com/EESSI/software-layer/pull/735/commits/b8555d1ee64330a0a43df838f4d1ec58a916a4b5

Reply:
If it should only happen when an actual install is being done, it belongs in post-install hook?

Reply:
Afraid not, that would cause the sanity check to (potentially) fail